src/components/Personalization/createViewStorage.js

```diff
@@ -0,0 +1,22 @@
+const viewStorage = {};
```

Comment:
When there are multiple instances, the viewStorage should be unique to the instance. You can achieve this by putting the viewStorage, addView, and storeViews declarations inside the default function.